Jenkinsfile

```diff
@@ -452,8 +452,6 @@ def pushCIDockerImages(Map args = [:]) {
       tagAndPush(beatName: 'osquerybeat', arch: arch, variants: defaultVariants)
     } else if ("${beatsFolder}" == "packetbeat"){
       tagAndPush(beatName: 'packetbeat', arch: arch, variants: defaultVariants)
-    } else if ("${beatsFolder}" == "x-pack/elastic-agent") {
-      tagAndPush(beatName: 'elastic-agent', arch: arch, variants: defaultVariants + completeVariant + cloudVariant)
```

Comment:
ditto -> remove https://github.com/elastic/beats/pull/30559/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R440-R442 which contains they definition for `completeVariant + cloudVariant`